packages/fuels-signers/src/provider.rs

```diff
@@ -232,6 +232,35 @@ impl Provider {
         Ok(coins)
     }
 
+    /// Get some spendable messages for address `from` that add up at least to
+    /// amount `amount`. The returned messages are actual messages that can be spent. The number
```

Comment:
IIUC, we cannot guarantee that the retrieved messages will add up to amount because we request `Resource`s and filter for messages later.

Are we sure that we want to go forward with this approach?

Reply:
We will talk about this at the meeting.